### PR TITLE
Add serverless framework boiler code

### LINKED ISSUE #1: Add boiler code for serverless framework in node and typescript

### PR DESCRIPTION
**Pull request by MosesSoftEng**
Closes #1

### CODE REVIEW

File: serverless.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import type { AWS } from '@serverless/typescript';
```
Comment (MosesSoftEng) [inline serverless.ts:1]: Serverless using javascript

File: src/functions/hello/index.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import schema from './schema';
```
Comment (MosesSoftEng) [inline src/functions/hello/index.ts:1]: Entry point

**Review by MosesSoftEng — COMMENTED** [review]
Good

**Review by MosesSoftEng — COMMENTED** [review]
Good

**Review by xngigez — APPROVED** [review]
Issue solved

**Pull request merged**